tests/utils/get-package-spec.ts

```diff
@@ -9,6 +11,15 @@ jest.mock('../../src/renderer/npm', () => ({
   findModulesInEditors: () => ['say'],
 }));
 
+describe('getForgeVersion', () => {
+  it('returns a semver version', () => {
```

Comment:
Test name nit

```suggestion
  it('returns a semver-compatible version constraint', () => {
```

Reply:
Good catch.  I was second-guessing myself on that right after I pushed